### ISSUE TITLE
Artificial time wait interval the 5.9.1 at the upload document step

### ISSUE THREAD

**Issue by aoumiri** [onfido/onfido-sdk-ui#1001]
### What was the expected behaviour?

Prior to the update, the upload document step was appearing without lag.

### What happened instead?

Now, there is a 2000 ms time interval after selecting the document type. It feels like a performance/network lag.

It also makes our unit tests fail because it exceeds timeouts.

To be noticed, on safari, the problem is not present, the page appears instantly. It's maybe related to this change: https://github.com/onfido/onfido-sdk-ui/compare/5.9.0...5.9.1#diff-3b5b2aa5b0b4b6d4703d57c8d79a43cbR17.

We had:

```js
      if (!isSafari131()) {
        this.cameraCheckerIntervalId = setInterval(this.checkCameraSupport, 2000)
      } else {
        this.checkCameraSupport()
      }
```

Maybe should be

```js
      if (!isSafari131()) {
        this.cameraCheckerIntervalId = setInterval(this.checkCameraSupport, 2000)
      }

      this.checkCameraSupport() // Do it right away regardless of the browser, to prevent the 2 seconds interval.
```

### Version info:

- Onfido SDK: 5.9.1
- Browser: chrome / firefox latest versions

### Integration configuration:

Not really relevant in this case, it's just an update, but I can add it if you wish, just ask :)

### Steps to reproduce:

Navigate until the upload document step, just before sending a POI. You should see the loader for exactly two seconds.

**Comment by kopijunkie** [comment]
Hi @aoumiri 

Thank you for raising this and the suggested fix. We'll be looking to get this fixed ASAP.

**Comment by kopijunkie** [comment]
Hi @aoumiri 

We have just released [5.9.2](https://github.com/onfido/onfido-sdk-ui/releases/tag/5.9.2) which fixes the issue now
